projects/elife.yaml

```diff
@@ -131,10 +131,24 @@ defaults:
             description: uses a plain Ubuntu basebox instead of an ami
             ec2:
                 ami: ami-9eaa1cf6 # Ubuntu 14.04
+        1604test:
+            description: uses Ubuntu 16.04 (Xenial) instead of 14.04 (Trusty)
+            ec2:
+                ami: ami-1d4e7a66 # xenial, build 20170811, hvm:ebs-ssd
+        standalone:
+            description: isolated from the master-server. requires access to the builder-private
+            ec2:
+                masterless: true
+        standalone1604:
+            description: isolated from the master-server and uses Ubuntu 16.04 (Xenial)
+            type: t2.small
+            ec2:
+                ami: ami-1d4e7a66 # xenial, build 20170811, hvm:ebs-ssd
+                masterless: true
     vagrant:
         # why 'bento'? https://bugs.launchpad.net/cloud-images/+bug/1569237
-        # box: bento/ubuntu-16.04 # Ubuntu 16.04 "Xenial"
-        box: ubuntu/trusty64 # Ubuntu 14.04
+        box: bento/ubuntu-16.04 # Ubuntu 16.04 "Xenial"
```

Comment:
16.04 as a default for all Vagrant VMs? Given rollout is going to be project-by-project, it should go at the project level until everything is ported
(or single projects could override this back with `ubuntu/trusty64` until they are porteD)

Reply:
I like the second option best. I introduced it because I was accidentally bringing up 14.04 instances because I missed a step updating the project file. as this introduces 16.04 support for all projects, I think I'll go through and add explicit 14.04 AMIs and boxes that we'll remove as we port each one

Reply:
done.